src/main/java/uk/ac/ebi/eva/pipeline/io/mappers/VariantVcfFactory.java

```diff
@@ -62,65 +64,27 @@
             throw new IllegalArgumentException("Not enough fields provided (min 8)");
         }
 
-        List<Variant> variants = new LinkedList<>();
-
-        String chromosome = fields[0];
+        String chromosome = removeChrPrefix(fields[0]);
         int position = Integer.parseInt(fields[1]);
```

Comment:
Shouldn't these be `getChromosome` (or `getChromosomeWithoutChrPrefix`) and `getPosition`, for the sake of consistency?